metador_core/widget/common.py

```diff
@@ -208,10 +210,16 @@ class Plugin(FileWidget.Plugin):
     @overrides
     def show(self) -> Viewable:
         return self.PANEL_WIDGET[self._meta.encodingFormat](
-            self.file_url(), width=self._w, height=self._h
+            self.file_url(),
+            width=self._w,
+            height=self._h,
+            alt_text="Sorry. Something went wrong while loading the resource",
         )
 
 
+# switching to HTML based audio/video panes because one from Panel not working
+# same problem in both case - url in str format throws an error
+# potential fixes from github not working
```

Comment:
if you refer to a known issue and have tried some solutions, consider adding links to relevant issues

This would allow to later stumble on this an see if there is some update on the issues changing the situation.